harness/tests/experiment/pytorch/test_pytorch_lightning.py

```diff
@@ -0,0 +1,102 @@
+import pathlib
+import typing
+from typing import Any, Dict
+
+import pytest
+
+import determined as det
+from determined import pytorch, workload
+from tests.experiment import utils  # noqa: I100
+from tests.experiment.fixtures import lightning_adapter_onevar_model as la_model
+
+
+def fork_trial_override_lm(overrides: dict):
```

Comment:
It seems overkilling to create dynamic classes on the fly. I am leaned towards creating new classes that inherit from `OneVarLM` and `OneVarTrial`?